src/components/ControlBar/styles/ConfirmDeleteDialog.module.css

```diff
@@ -0,0 +1,4 @@
+.content {
```

Comment:
Do we have this style somewhere centralised to be reused? Seems like regular text to me, isn't that already defined by the style guide and implemented somewhere else already?

Reply:
Yes, a little strange but I didn't find anything in ui-core. Typography is described in the design docs, but there is nothing implemented in ui-core yet.

Unfortunately there is css from ext.js that is setting the font size and line height which is affecting the modal content, so it had to be overridden, for now. I'll mention it to the ui team and revisit this.